packages/tools/devtools/devtools-view/src/DevtoolsView.tsx

```diff
@@ -464,14 +466,17 @@ function Menu(props: MenuProps): React.ReactElement {
 
 	function onTelemetryClicked(): void {
 		setSelection({ type: "telemetryMenuSelection" });
+		usageLogger?.sendTelemetryEvent({ eventName: "TelemetryMenuSectionClicked" });
```

Comment:
I'm honestly still debating with myself about this but I'm leaning towards having a single eventName "Navigation" with details specifiying where/what kind. Particularly, to encompass view changes due to menu options being clicked, or tabs inside a given view being clicked.

I.e. (only one of the `detail.target` values would apply to a given event):

```plaintext
{
  eventName: "Navigation",
  details: { target: "Menu_Home" / "Menu_Container" / "Menu_Telemetry" / "Menu_Settings" / "Container_<name>Tab" }
```

Can we make that change? It would affect the menu options here and the tabs in the Container view.